src/Unosquare.Labs.EmbedIO/Extensions.cs

```diff
@@ -533,5 +456,18 @@ public static byte[] Compress(this byte[] buffer, CompressionMethod method = Com
         }
 
         #endregion
+
+        internal static T GetValueOrDefault<T>(this Dictionary<T, T> dict, T key, T defaultValue = default(T))
+        {
+            return dict.ContainsKey(key) ? dict[key] : defaultValue;
+        }
+        
+        internal static void ForEach<T>(this Dictionary<T, T> dict, Action<T, T> mapAction)
```

Comment:
shouldn't be called mapAction..maybe itemAction

Reply:
agree